mesonbuild/linkers/linkers.py

```diff
@@ -1316,7 +1316,8 @@ def __init__(self, for_machine: mesonlib.MachineChoice, always_args: T.List[str]
                          prefix, always_args, machine=machine, version=version, direct=direct)
 
     def get_always_args(self) -> T.List[str]:
-        return self._apply_prefix(['/nologo', '/release']) + super().get_always_args()
+        parent = super().get_always_args()
+        return parent + self._apply_prefix(['/release'])
```

Comment:
What's the reason for breaking this across two lines instead of deleting one word from the existing line?

Reply:
I did like in VisualStudioLikeLinkerMixin. Otherwise, there are no reasons.

Reply:
Please just remove it from the list, that's simpler :)